### PR TITLE
Usermanagement static data resolved

### PR DESCRIPTION
**Pull request by Abhinav4952**
Resolved all comments for user management static data

### CODE REVIEW

File: AngularTraining/user-management-staticdata/src/app/app.component.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="login-name">{{loggedinUser}}</div>
+<div class="nav-bar">
+  <div><a routerLink="/">Home</a></div>
+  <div routerLinkActive="active"><a routerLink="feature1">Deleted</a></div>
+  <div routerLinkActive="active"><a routerLink="feature2">Active</a></div>
+  <div routerLinkActive="active"><a routerLink="feature3">Manage</a></div>
+</div>
+
+<hr style="width: 90%; margin-top: 5px; margin-bottom: 5px; border: none;border-top: 1px solid gainsboro;" />
```
Comment (vishal8885) [inline AngularTraining/user-management-staticdata/src/app/app.component.html:9]: don't use inline styling

File: AngularTraining/user-management-staticdata/src/app/feature3/UserInfo/UserInfo.component.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { UsersService } from 'src/app/users.service';
+
+@Component({
+  selector: 'app-userinfo',
+  templateUrl: './f3c2.component.html',
```
Comment (vishal8885) [inline AngularTraining/user-management-staticdata/src/app/feature3/UserInfo/UserInfo.component.ts:7]: template name should be app-userinfo.component.html

File: AngularTraining/user-management-staticdata/src/app/app.component.html (hunk 1 above)
Comment (Abhinav4952) [reply AngularTraining/user-management-staticdata/src/app/app.component.html]: Removed inline styling

File: AngularTraining/user-management-staticdata/src/app/feature3/UserInfo/UserInfo.component.ts (hunk 2 above)
Comment (Abhinav4952) [reply AngularTraining/user-management-staticdata/src/app/feature3/UserInfo/UserInfo.component.ts]: Made changes to all *.html and *.css

**Pull request merged**